sw/airborne/boards/disco.h

```diff
@@ -53,10 +53,12 @@
 #endif
 
 /* Default actuators driver */
+//#ifdef BOARD_DISCO
```

Comment:
please remove unneeded commented code from this file

Reply:
OK